api/src/LinkyLink/Controllers/LinksController.cs

```diff
@@ -87,6 +87,24 @@ public async Task<ActionResult<LinkBundle>> GetLinkBundlesForUser(string userId)
         /// <summary>
         /// Create one or more LinkBundles.
         /// </summary>
+        /// <remarks>
```

Comment:
I think its not necessary  to provide an example here for the payload. There is already a payload example present in existing json files and validate-api.http

Reply:
I placed the annotation on there mainly for the Swagger UI.  In the Swagger UI the user will see the request body schema denoted in annotation and can use it to contact the end point.